config/mappings_hash.rb

```diff
@@ -30,7 +30,7 @@
   'physical_description_form'       => '//physicalDescription/form/text()', # .to_s
   'physical_description_media_type' => '//physicalDescription/internetMediaType/text()', # .to_s
 
-# location
+# physical_location
```

Comment:
I think this is a defunct file (I made an issue ticket for it and put it in Ready column in Waffle board)